general-concepts/dependencies/text.xml

```diff
@@ -410,6 +410,35 @@ RDEPEND="media-libs/cogl:1.0=
 	gnutls? ( &gt;=net-libs/gnutls-2.8:= )"
 </codesample>
 
+<p>
+means that only the '1.0' slot is acceptable for <c>media-libs/cogl</c> and
+that sub-slot changes of <c>media-libs/cogl</c> will cause a rebuild of the
+dependant package. And that every slot of <c>net-libs/gnutls</c> is
+acceptable but here to, any slot change is causing a rebuild.
+</p>
+
+<p>
+The <c>:slot</c> dependency syntax continues to behave like in EAPI=4 or
```

Comment:
Isn't this example pretty much the same as the previous example?

Reply:
Thanks for the feedback. :) Unfortunately I am not sure if I can follow. The previous example deals with the the `:SLOT=` and `:=` syntax. While this is about the `:SLOT` syntax. Am I missing something here?

Reply:
No, I just think the difference isn't significant enough to warrant a unique example. But I don't care much either way :)